Makefile

```
@@ -1,5 +1,15 @@
 .PHONY: all binary dynbinary build cross help install manpages run shell test test-docker-py test-integration test-unit validate win
 
+ifdef USE_BUILDX
+BUILDX ?= $(shell which buildx)
```

nit; shellcheck recommends using `command -v`, which is the POSIX approach https://github.com/koalaman/shellcheck/wiki/SC2230

```suggestion
BUILDX ?= $(shell command -v buildx)
```